Address reviews (#19)
mihaibudiu/sigmod-rh: vldb-best/experiments.tex

the central block, the \dbsp query engine.  However, in most real-life
application performance will be limited by the adapters and network.

\subsection{Cost of incremental updates}

Here we validate our claim that updates in \dbsp have a cost
proportional to the size of the change.

> We ran TPC-H on a desktop computer, with a scale factor 100; the input

Reviewer: ```suggestion
We ran TPC-H on a desktop computer, with a scale factor 100, meaning that the input
```

data is about 100 GB, consisting of about 1.6 billion records.  We
divided the largest tables, \emph{orders} and \emph{lineitem}, which
in total contain about half the input records, into 10 equal-sized
batches, each comprising about 75,000,000 records.  Because
\emph{orders} records refer to \emph{lineitem} records, we ensured
that if an \emph{order} record was in a batch, so were its
\emph{lineitem} records.  Then, for each batch in turn, we inserted
its records.

> We run this computation using two database systems: (1) using a

Reviewer: ```suggestion
We ran this computation using two database systems: (1) using a
```

service performance and configuration as a possible source of
variability.

Figure~\ref{fig:macrobenchmark} reports our measurements.  We show 3
bars for \dbsp: the first (blue) is using hand-written Rust code based
on the \dbsp runtime, running in memory; the second (orange) bar is
written in SQL and runs in memory, while the third (red) bar shows the
SQL program using secondary storage.

\paragraph{Throughput.}

\newcommand{\x}{\(\times\)}

Figure~\ref{fig:macrobenchmark}(a) shows the normalized throughput of
\dbsp versus Flink (which is always 1).  With storage disabled, \dbsp
is up to 17\x{} faster than Flink, with a geometric mean of 2.2\x{}
faster on average.  As queries get more complicated, \dbsp's advantage
over Flink grows by a much larger factor.  We also notice that there

> is still a roughly 2x gap to cover between the performance of

Reviewer: ```suggestion
is still a roughly 2\x{} gap to cover between the performance of
```
